R/conversion.R

```diff
@@ -160,7 +160,29 @@ py_to_r.pandas.core.series.Series <- function(x) {
   py_to_r(x$as_matrix())
 }
 
+pandas_shape <- function(object) unlist(as_r_value(object$shape))
 
+#' @export
+summary.pandas.core.series.Series <- function(object, ...) {
+  if (py_is_null_xptr(object) || !py_available())
+    str(object)
+  else
+    object$describe()
+}
+
+#' @export
+length.pandas.core.series.Series <- function(x) {
+  if (py_is_null_xptr(x) || !py_available())
+    0L
+  else {
+    pandas_shape(x)[[1]]
+  }
+}
+
+#' @export
+dim.pandas.core.series.Series <- function(x) {
```

Comment:
Just want to confirm that the `NULL` return here is intentional

Reply:
Yes, see my comment below.

Reply:
@jjallaire Travis is having issues. Could you re-start it (I don't have write access to this repo)?